build.sbt

```
@@ -21,7 +21,8 @@ publishLocal / skip := true
   */
 val suppressSemVerCheckOfNewModuleUntilNextVersion = semVerCheck := {
   version.value match {
-    case VersionNumber(Seq(1, 3, 0 | 1, _*), _, _) => Def.task {}
+    case VersionNumber(Seq(1, x, _*), _, _) if x <= 3 => Def.task {}
+    case VersionNumber(Seq(1, 4, 0, _*), _, _) => Def.task {}
```

Q: Why isn't this:
```suggestion
    case VersionNumber(Seq(1, x, _*), _, _) if x <= 4 => Def.task {}
```
? 

If there's a reason I'd document it, because it looks unnecessary to split them.

---

It's because running the tests locally will fail if the version is 1.3.x (which is the current state on master). I also need to merge this and tag it with 1.4.0, so I need to ignore that version to publish this library. But when I hit 1.4.1, I want this to fail the build.